.github/CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+#
+# Description:
+#  Code ownership file for the okta/okta-help GitHub repository for
+#  mapping resources to teams.
+#  See [CODEOWNERS] for details.
+#
+# Notes:
+#  Users must have write access to the repo in order for this mechanism to
+#  work correctly.
+#  This will likely be true of @okta/team but not @team-okta
+#
+# Resources
+#  - [CODEOWNERS] https://help.github.com/articles/about-codeowners
+#
+
+###############################################################################
+# Aliasing utility
```

Comment:
nit: if we want to "group" what files are used for what purpose, I would prefer something like
```
# Aliasing utility
/i18n-utils/
/okta_help.html
# End aliasing utility
```
This way we prevent in the future people adding lines entries in the wrong "section"